app/views/agents_profile/index.html.haml

```diff
@@ -0,0 +1,9 @@
+.container.d-flex.flex-column.py-4{ style: "width: 1248px;" }
```

Comment:
instead  of writing `style: "width: 1248px;"` all over create a css class that you reuse